utils/make_symlinks.bash

```diff
@@ -18,11 +18,61 @@
 # Unknown option or missing argument: 'f' ignored
 
 ### Cryosphere ###
-hsi ln -s /home/d/dcomeau/cryosphere_simulations/20190430.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.control.cori-knl /home/projects/e3sm/www/Cryosphere/E3SMv1/20190430.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.control.cori-knl
-for simulation in "20190306.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.edison" "20190923.GMPAS-IAF.T62_oEC60to30v3wLI.cori-knl" "20190225.GMPAS-DIB-IAF-ISMF.T62_oEC60to30v3wLI.cori-knl" "20200610.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_ECwISC30to60E1r2.cori-knl.maint1p2-3DGM" "20190819.GMPAS-DIB-IAF-ISMF.T62_oEC60to30v3wLI.cori-knl.testNewGM" "20210614.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.DIBbugfix.anvil" "20210614.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_ECwISC30to60E1r2.anvil.DIBbugFixMGM"; do
-    hsi ln -s /home/projects/m3412/${simulation} /home/projects/e3sm/www/Cryosphere/E3SMv1/${simulation}
-done
+# hsi ln -s /home/d/dcomeau/cryosphere_simulations/20190430.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.control.cori-knl /home/projects/e3sm/www/Cryosphere/E3SMv1/20190430.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.control.cori-knl
+# for simulation in "20190306.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.edison" "20190923.GMPAS-IAF.T62_oEC60to30v3wLI.cori-knl" "20190225.GMPAS-DIB-IAF-ISMF.T62_oEC60to30v3wLI.cori-knl" "20200610.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_ECwISC30to60E1r2.cori-knl.maint1p2-3DGM" "20190819.GMPAS-DIB-IAF-ISMF.T62_oEC60to30v3wLI.cori-knl.testNewGM" "20210614.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_oECv3wLI.DIBbugfix.anvil" "20210614.A_WCYCL1850-DIB-ISMF_CMIP6.ne30_ECwISC30to60E1r2.anvil.DIBbugFixMGM"; do
+#     hsi ln -s /home/projects/m3412/${simulation} /home/projects/e3sm/www/Cryosphere/E3SMv1/${simulation}
+# done
 
-# TODO: complete the v1 cryo csv
-# TODO: add docs for v1 cryo
-# TODO: run the table generator script for that csv
+### BGC ###
+# https://e3sm.atlassian.net/wiki/spaces/ED/pages/4495441922/V1+Simulation+backfill+WIP
+# We'll use symlinks to the publication archives
+declare -A simulation_map=(
+    # CTC Control
+    ["20181217.CNTL_CNPCTC1850_OIBGC.ne30_oECv3.edison"]="pub_archive_E3SM_1_1_piControl"
+    ["20191204.CNTL_CNPCTC1850_OIBGC.ne30_oECv3.compy"]="pub_archive_E3SM_1_1_piControl-ext85yr"
+    # CTC BCRC
+    ["20181217.BCRC_CNPCTC20TR_OIBGC.ne30_oECv3.edison"]="pub_archive_E3SM_1_1_hist-BCRC"
+    ["20191107.BCRC_CNPCTC_SSP585_OIBGC.ne30_oECv3.compy"]="pub_archive_E3SM_1_1_ssp585-BCRC"
+    # CTC BCRD
+    ["20181217.BCRD_CNPCTC20TR_OIBGC.ne30_oECv3.edison"]="pub_archive_E3SM_1_1_hist-BCRD"
+    # These 2 map to the same pub archive:
```

Comment:
There are 3 instances in https://e3sm.atlassian.net/wiki/spaces/ED/pages/4495441922/V1+Simulation+backfill+WIP where _two_ simulation names map to the same publication archive (this appears to be because the run was split into two different runs to cover different years, perhaps running on different days or different machines). 

I assumed we wanted the long-form names like  `BCRC_CNPECACNT_20TR.ne30_oECv3.edison` and `BCRC_CNPECACNT_20TR.ne30_oECv3.cori-knl`, in which case it appears I'd have to symlink to the original directories rather than the publication archive. (I wanted to symlink to the publication archive because that would be less likely to be deleted than user-owned directories).